packages/openapi-generator/src/cli/index.ts

```diff
@@ -101,13 +100,8 @@ class GenerateOpenApiClient extends Command {
   };
 
   async run(): Promise<void> {
-    try {
-      const parsed = this.parse(GenerateOpenApiClient);
-      await generate(parsed.flags);
-    } catch (e) {
-      logger.error(e.message);
-      return cli.exit(1);
-    }
+    const parsed = this.parse(GenerateOpenApiClient);
+    await generate(parsed.flags);
```

Comment:
Please correct me. 
In the `generate()`, potentially an error will be thrown, which is not handled in this line.
This should result in `UnhandledPromiseRejectionWarning` instead of "exit 1" as the old behaviour.
Is this intended.

Reply:
Actually no. In the CLI this will be all handled through the run script which already handles errors. However, I reverted this back, because our logger basically makes the error red on the command line and this currently goes nicer together with our other logs. Let's reconsider this for the verbose PR.